FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/V2ListItemActivity.kt

```diff
@@ -375,25 +390,35 @@ private fun OneLineListAccessoryContentContent(context: Context) {
         )
         ListItem.Item(
             text = primaryText,
+            onClick = {},
+            leadingAccessoryContent = { LeftContentAvatar(size = Size24) },
+            border = BorderType.Bottom,
```

Comment:
not sure if this was part was intended as it will lead to UI change of tester. 

if yes, can you add screenshot for this change in the PR description ?

Reply:
Change is only for onClick. I guess git diff is messing it up with below ListItem.

Reply:
Yes, added only onClick={}